submission-scripts/script-templates/Makefile

```diff
@@ -622,3 +624,24 @@ define create-storm
 	sed -i -e "s/ZOOKEEPER_SETUP=.*/ZOOKEEPER_SETUP=yes/" $(MAGPIE_STORM)
 	sed -i -e "s/ZOOKEEPER_VERSION=\"\(.*\)\"/ZOOKEEPER_VERSION=\"$(STORM_ZOOKEEPER_VERSION_DEFAULT)\"/" $(MAGPIE_STORM)
 endef
+
+define create-zeppelin
+	echo "Creating magpie.$(SCHED)-zeppelin"
+	cat 	magpie-header \
+		magpie-config-$(SCHED) \
+		magpie-magpie-customizations-substitution \
+		magpie-general-configuration \
+		magpie-general-configuration-python \
+		magpie-spark \
+		magpie-zeppelin \
+		magpie-run-job-header-substitution \
+		magpie-run-job-$(DIST) > $(MAGPIE_ZEPPELIN)
+	$(call common-substitution, ${MAGPIE_ZEPPELIN})
+	$(call common-additions, ${MAGPIE_ZEPPELIN}, zeppelin)
+	sed -i -e "s/MAGPIE_JOB_TYPE=\"\(.*\)\"/MAGPIE_JOB_TYPE=\"spark\"/" $(MAGPIE_ZEPPELIN)
+	sed -i -e "s/SPARK_SETUP=.*/SPARK_SETUP=yes/" $(MAGPIE_ZEPPELIN)
+	sed -i -e "s/^# export SPARK_LOCAL_SCRATCH_DIR=/export SPARK_LOCAL_SCRATCH_DIR=/g" $(MAGPIE_ZEPPELIN)
+    sed -i -e 's/export SPARK_MODE="\(.*\)"/export SPARK_MODE="setuponly"/' $(MAGPIE_ZEPPELIN)
```

Comment:
Zeppelin actually starts the spark cluster itself. If you start Spark you will end up with two competing clusters.